Tasks/KubernetesManifestV1/src/clusterconnection.ts

```diff
@@ -38,7 +38,12 @@ export default class ClusterConnection {
             const azureSubscriptionEndpoint : string = tl.getInput("azureSubscriptionEndpoint", true);
             const resourceGroup : string = tl.getInput("azureResourceGroup", true);
             const useClusterAdmin: boolean = tl.getBoolInput('useClusterAdmin');
-            return this.loadClusterType(connectionType).getKubeConfig(azureSubscriptionEndpoint, resourceGroup, clusterName, useClusterAdmin)
+            const resourceType: string = (tl.getInput("resourceType", false) || "Microsoft.ContainerService/managedClusters");
+            if (resourceType.toLowerCase() == "microsoft.containerservice/fleets"){
+                return this.loadClusterType(connectionType).getKubeConfigForFleet(azureSubscriptionEndpoint, resourceGroup, clusterName, useClusterAdmin)
```

Comment:
I think this won't get compiled or work.  I believe the new Fleet function doesn't have `useClusterAdmin` parameter.